tests/arg_parse/test_arg_parse_version_check.h

```diff
@@ -0,0 +1,332 @@
+// ==========================================================================
+//                                 arg_parse
+// ==========================================================================
```

Comment:
// ==========================================================================
//                 SeqAn - The Library for Sequence Analysis
// ==========================================================================